src/assessments/seo/metaDescriptionLengthAssessment.js

```diff
@@ -2,7 +2,9 @@ let AssessmentResult = require( "../../values/AssessmentResult.js" );
 let Assessment = require( "../../assessment.js" );
 let merge = require( "lodash/merge" );
 
-const maximumLength = 320;
+import Config from "../../config/config";
+
+const MAXIMUM_META_DESCRIPTION_LENGTH = Config.maxMeta;
```

Comment:
Is there a specific reason for this new naming convention for variables? (all caps with underscores instead of camelcase)

Reply:
We don't really have a convention for that, and since the value was stored in two differently named variables in `metaDescriptionLengthAssessment.js` and `snippetPreview.js` I used all caps because of personal preference.

Reply:
Ah, but now the value is just imported from the config. So I think it would make sense to stick with the general naming convention in YoastSEO.js and use camel case.